tests/unit/internal-session-test.js

```diff
@@ -18,7 +18,8 @@ describe('InternalSession', () => {
     store         = EphemeralStore.create();
     authenticator = Authenticator.create();
     session       = InternalSession.create({ store, container });
-    sinon.stub(container, 'lookup').withArgs('authenticator').returns(authenticator);
+    sinon.stub(container, 'lookup').withArgs('authenticator').returns(authenticator)
+      .withArgs('authenticator:test').returns(authenticator);
```

Comment:
This should actually never be called with `'authenticator:test'` in this test. I think the actual problem isn't that this stub was missing but that some lookup is made that isn't supposed to be made. Some previous test might not be cleaning up the session store correctly. I assume that the test passes in isolation.